### PR TITLE
Update OBO Dashboard Maintainer People

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

**Comment by anitacaron** [comment]
I don't have write access, so I cannot ask for a reviewer, @matentzn.

File: _data/roles.yml [hunk 1]
```diff
@@ -68,9 +68,10 @@
     Working with OBO Ops to find people for extending dashboard when new principles come along.
   responsibilities:
   people:
-    - name: matentzn
```
[the remark this answers is not in the document]
Comment (matentzn) [reply _data/roles.yml:71]: You mean to document previous role holders? Why not I guess.

[the remark this answers is not in the document]
Comment (pfabry) [reply _data/roles.yml:71]: In addition, what is the function of the "open" tag? Should it be set to false once new role are filled ?

[the remark this answers is not in the document]
Comment (matentzn) [reply _data/roles.yml:71]: `open` means that the position is not filled, you can either delete this or set it to false.

**Review by matentzn — APPROVED** [review]

**Pull request merged**